src/components/Pensjonssimulering/utils.ts

```diff
@@ -39,31 +39,14 @@ export const PENSJONSGIVENDE_DATA = [
   650000, 260000, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 ]
 
-export const AFP_DATA = [
-  0, 20000, 80000, 80000, 80000, 80000, 80000, 80000, 80000, 80000, 80000,
-  80000, 80000, 80000, 80000, 80000, 80000, 80000,
-]
-
-export const FOLKETRYGDEN_DATA = [
-  0, 35000, 175000, 175000, 175000, 175000, 175000, 175000, 175000, 175000,
-  175000, 175000, 175000, 175000, 175000, 175000, 175000, 175000,
-]
-
-export const simulateDataArray = (
-  array: number[],
-  length: number,
-  startAge?: number,
-  coefficient = 0
-) => {
-  if (startAge && startAge < 60) {
-    throw Error("Can't simulate dataArray when startAge is smaller than 60")
-  }
-  const faktor = startAge ? startAge - 60 : 0
-  const dataArray = [...array].map((value, i) => {
-    return i > 1 ? value + faktor * coefficient : value
+export const processPensjonsberegningArray = (
+  pensjonsberegninger: Pensjonsberegning[] = []
```

Comment:
TODO: Her bør det legges startAlder og sluttAlder, ved f.eks bruk av MAX_UTTAKSALDER slik at array'ene for alderspensjon, afp og tjenestepensjon alltid har lik lengde. Hvis ikke vil Highcharts vise uheldige resultater